server/sandbox_stop_linux.go

```diff
@@ -55,7 +56,7 @@ func (s *Server) stopPodSandbox(ctx context.Context, sb *sandbox.Sandbox) error
 				}
 				c := ctr
 				waitGroup.Go(func() error {
-					if err := s.StopContainerAndWait(ctx, c, int64(10)); err != nil {
+					if err := s.StopContainerAndWait(ctx, c, timeout); err != nil {
```

Comment:
Instead of a 10 seconds hardcoded timeout to stop containers,  we will have now a configured timeout, with a value of 10. How will this speed up actually the crio wipe ? Should  end userss change the value in the crio config too ?

Reply:
The timeout shouldn't need to be configured for crio wipe

If the node rebooted (which is the likliest reason the pod can't be restored) then there's no process to stop, so this is basically just updating runtime metadata

If just crio restarted, and there was some issue with the restore, then we likely want to restart the pod asap to allow for a new pod to be created that is not in this state. Note: I would consider a failure to restore a rare event